src/app/public/modules/action-button/types/action-button-container-align-items.ts

```diff
@@ -0,0 +1,13 @@
+export enum SkyActionButtonContainerAlignItems {
```

Comment:
Making this a `const enum` will reduce the bundle size during an AoT build.

Reply:
Done.